SmartEVSE-3/src/evse.cpp

```diff
@@ -2110,6 +2164,210 @@ uint8_t PollEVNode = NR_EVSES;
+    {
+        uint16_t RequestedCurrent = payload.toInt();
+        if (RequestedCurrent == 0)
+        {
+            OverrideCurrent = 0;
+        }
+        else if (Mode == MODE_NORMAL || Mode == MODE_SMART)
+        {
+            if (RequestedCurrent >= (MinCurrent * 10) && RequestedCurrent <= (MaxCurrent * 10))
+            {
+                OverrideCurrent = RequestedCurrent;
+            }
+        }
+    }
+    else if (topic == MQTTprefix + "/Set/MainsMeter")
+    {
+        if (MainsMeter != EM_API)
+            return;
+
+        int32_t L1, L2, L3;
+        int n = sscanf(payload.c_str(), "%d:%d:%d", &L1, &L2, &L3);
+
+        if (n == 3 && L1 < 1000 && L2 < 1000 && L3 < 1000)
+        {
+            phasesLastUpdate = time(NULL);
+
+            if (LoadBl < 2)
+                timeout = 20;
+            if ((ErrorFlags & CT_NOCOMM))
+                ErrorFlags &= ~CT_NOCOMM; // Clear communication error, if present
+
+            Irms[0] = L1;
+            Irms[1] = L2;
+            Irms[2] = L3;
+
+            int batteryPerPhase = getBatteryCurrent() / 3;
+            Isum = 0;
+            for (int x = 0; x < 3; x++)
+            {
+                IrmsOriginal[x] = Irms[x];
+                Irms[x] -= batteryPerPhase;
+                Isum = Isum + Irms[x];
+            }
+
+            UpdateCurrentData();
+        }
+    }
+    else if (topic == MQTTprefix + "/Set/EVMeter")
+    {
+        if (EVMeter != EM_API)
+            return;
+
+        int32_t L1, L2, L3, W, WH;
+        int n = sscanf(payload.c_str(), "%d:%d:%d:%d:%d", &L1, &L2, &L3, &W, &WH);
+
+        // We expect 5 values (and accept -1 for unknown values)
+        if (n == 5)
+        {
+            if ((L1 > -1 && L1 < 1000) && (L2 > -1 && L2 < 1000) && (L3 > -1 && L3 < 1000))
+            {
+                // RMS currents
+                Irms_EV[0] = L1;
+                Irms_EV[1] = L2;
+                Irms_EV[2] = L3;
+
+                UpdateCurrentData();
+            }
+
+            if (W > -1)
+            {
+                // Power measurement
+                PowerMeasured = W;
+            }
+
+            if (WH > -1)
+            {
+                // Energy measurement
+                EnergyEV = WH;
+                if (ResetKwh == 2)
+                    EnergyMeterStart = EnergyEV;             // At powerup, set EnergyEV to kwh meter value
+                EnergyCharged = EnergyEV - EnergyMeterStart; // Calculate Energy
+                RecomputeSoC();                              // Recalculate SoC
+            }
+        }
+    }
+    else if (topic == MQTTprefix + "/Set/HomeBatteryCurrent")
+    {
+        homeBatteryCurrent = payload.toInt();
+        homeBatteryLastUpdate = time(NULL);
+    }
+
+    // Make sure MQTT updates directly to prevent debounces
+    lastMqttUpdate = 10;
+}
+
+void SetupMQTTClient()
+{
+    // Disconnect existing connection if connected
+    if (MQTTclient.connected())
+    {
```

Comment:
In these "simple", unambiguous if statements, please leave out the { and }, saves lines and make code more readable.
So:
if (MQTTclient.connected())
      MQTTclient.disconnect();

Reply:
Same as above actually; but I'll make it so.